lib/App.php

```diff
@@ -57,7 +53,7 @@ public function notify(INotification $notification): void {
 		try {
 			$this->push->pushToDevice($notificationId, $notification);
 		} catch (NotificationNotFoundException $e) {
-			throw new \InvalidArgumentException('Error while preparing push notification');
+			$this->logger->error('Error while preparing push notification', ['exception' => $e]);
```

Comment:
Not throwing anything anymore? What will happen to the notification, is it removed from queue?

Reply:
There is no queue. Result is that it's now logged here, instead of silently ignored in the manager that iterates over all registered apps